### PR TITLE
Document DDoS mitigation

### LINKED ISSUE #3089: Document DDoS mitigation
The `BLOCKED_IPS` env variable is not documented here:
https://mozilla-pontoon.readthedocs.io/en/latest/admin/deployment.html.

We should also add a paragraph here on DDoS mitigation:
https://mozilla-pontoon.readthedocs.io/en/latest/admin/maintenance.html

### PR DESCRIPTION
**Pull request by mathjazz**
Fix #3089.

### CODE REVIEW

File: docs/admin/maintenance.rst [hunk 1]
```diff
@@ -40,3 +40,16 @@ Finally, you need to simply access the worker:
...
    # Replace my-app-name with your Heroku app's name.
    celery --broker=`heroku config:get RABBITMQ_URL --app=my-app-name` worker
+
+Mitigating DDoS attacks
+-----------------------
+In a distributed denial-of-service attack (`DDoS`_ attack), the incoming traffic
+flooding the victim originates from many different sources. This stops everyone
+else from accessing the website as there is too much traffic flowing to it.
+
+One way to mitigate DDoS attacks is to identify the IP addresses of the
```
[the remark this answers is not in the document]
Comment (mathjazz) [reply docs/admin/maintenance.rst:50]: Heroku?

[the remark this answers is not in the document]
Comment (flodolo) [reply docs/admin/maintenance.rst:50]: Yes, I have a feeling I've seen the IP from the machine itself in the logs, and I'm terrified of blocking the wrong IP (I run a whois…).

[the remark this answers is not in the document]
Comment (mathjazz) [reply docs/admin/maintenance.rst:50]: I wouldn't know. But yes, with BLOCKED_IPS you can also block a bad IP address. Not sure we can do much about it?

[the remark this answers is not in the document]
Comment (flodolo) [reply docs/admin/maintenance.rst:50]: As long as I'm not blocking Pontoon itself (e.g. request between workers or db).

**Review by flodolo — APPROVED** [review]

**Pull request merged**